etl/src/energuide/embedded/heating.py

```diff
@@ -82,11 +82,11 @@ def _get_output_size(cls, node: element.Element) -> float:
         capacity: typing.Optional[float]
         if units == 'kW':
             capacity = capacity_value
-        elif units == 'btu/h':
+        elif units == 'btu/hr':
             capacity = capacity_value / cls._KWH_TO_BTU
         else:
             raise InvalidEmbeddedDataTypeError(
-                Heating, f'Unknown capacity units: f{units}')
+                Heating, f'Unknown capacity units: {units}')
```

Comment:
The above two bugs had me googling for what the heck a `fbtu/hr` was. It showed up in my stacktrace, so yay logging!